spaces/models/space.py

```diff
@@ -60,7 +60,7 @@ class Meta:
     def __str__(self) -> str:
         value = self.name
         if self.unit is not None:
-            value += f", {self.unit.name}"
```

Comment:
I think the `!s` here is redundant, as wouldn't`{self.unit}` just call the str() function on the `unit`, which is the same as `!s`? 🤔